error_handler.py

```
@@ -0,0 +1,273 @@
+    if isinstance(error, (discord.HTTPException, asyncio.TimeoutError)):
+        return True
+    
+    # Database busy errors
+    if isinstance(error, sqlite3.OperationalError) and "database is locked" in str(error).lower():
+        return True
+    
+    # Rate limiting (though should be handled differently)
+    if isinstance(error, discord.HTTPException) and error.status == 429:
+        return True
+    
+    return False
```

Ruff (0.11.9)

Return the condition directly

Inline condition

(SIM103)